nimpretty/tests/expected/exhaustive.nim

```diff
@@ -686,8 +686,7 @@ block:
 proc newRecordGen(ctx: Context; typ: TypRef): PNode =
   result = nkTypeDef.t(
     newId(typ.optSym.name, true, pragmas = [id(
-        if typ.isUnion: "cUnion" else: "cStruct")]),
-    empty(),
+    if typ.isUnion: "cUnion" else: "cStruct")]), empty(),
```

Comment:
but this is worse

Reply:
Yes, it is worse, but it wasn't human-pretty even before. I took it as positive trade-off: slightly worse here, but it makes other code much nicer.